### DISCUSSION TITLE
Services listing and grid view

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#22] [General]
As per the DLS 2.0 template, services directory should have 2 views (grid/listing) however the category filter and search have different style and position on each template. Our initial understanding is that the user will browse the services directory page then have the ability to switch between the two different views, but that is possible only if the search and category filter have the same placement and style. please advise us on how to implement this or if it fine to use only one of them
Another suggestion is to create two separate services directory for each view

**Comment by moiz-conceptlz** [comment]
Hi @ahmedabbous 

Lets consider an example of something like this page - https://designsystem.gov.ae/docs/components

The page is divided first into 2 section, which has a side bar and a content area. The content area is further divided into a grid layout of 3 items per row.

It is our suggestion to utilize a similar approach where the filter is the sidebar, and the content area can have a grid layout for a card approach and turn it into a flex layout for a list view.  It is not recommended to create a separate page itself to display a list and grid view. The switch between them must be on the space page - its only a preference on how a user may want to view the list of services.

UX studies also show that when a filter is utilized, a list view is considered easier for a user to locate information.
